src/general/corrections.jl

```diff
@@ -71,21 +65,6 @@ to an improvement, especially at free surfaces.
 !!! note
     - It is also referred to as "0th order correction".
     - In 2D, we can expect an increase of about 5--6% in computation time.
-
-
-## References
-- J. Bonet, T.-S.L. Lok.
-  "Variational and momentum preservation aspects of Smooth Particle Hydrodynamic formulations".
-  In: Computer Methods in Applied Mechanics and Engineering 180 (1999), pages 97-115.
-  [doi: 10.1016/S0045-7825(99)00051-1](https://doi.org/10.1016/S0045-7825(99)00051-1)
-- Mihai Basa, Nathan Quinlan, Martin Lastiwka.
-  "Robustness and accuracy of SPH formulations for viscous flow".
-  In: International Journal for Numerical Methods in Fluids 60 (2009), pages 1127--1148.
-  [doi: 10.1002/fld.1927](https://doi.org/10.1002/fld.1927)
--  Shaofan Li, Wing Kam Liu.
-  "Moving least-square reproducing kernel method Part II: Fourier analysis".
-  In: Computer Methods in Applied Mechanics and Engineering 139 (1996), pages 159--193.
-  [doi:10.1016/S0045-7825(96)01082-1](https://doi.org/10.1016/S0045-7825(96)01082-1)
```

Comment:
These references are important. At least Bonet & Lok is the standard source for this.
CC @svchb

Reply:
It is not mentioned in the description, so Idk where to put the references.